tests/Unit/Rpc/Handler/OverloadMethodHandlerTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Phpactor\Tests\Unit\Rpc\Handler;
+
+use PHPUnit\Framework\TestCase;
+use Phpactor\Tests\Unit\Rpc\Handler\HandlerTestCase;
+use Phpactor\WorseReflection\Reflector;
+use Phpactor\WorseReflection\Core\SourceCodeLocator\StringSourceLocator;
+use Phpactor\Rpc\Handler\OverloadMethodHandler;
+use Phpactor\Rpc\Handler;
+use Phpactor\WorseReflection\Core\Logger\ArrayLogger;
+use Phpactor\WorseReflection\Core\SourceCodeLocator;
+use Phpactor\WorseReflection\Core\SourceCode;
+use Phpactor\Rpc\Response\Input\ChoiceInput;
+use Phpactor\CodeTransform\Domain\Refactor\OverloadMethod;
+use Phpactor\CodeTransform\Domain\SourceCode as TransformSourceCode;
+use Phpactor\Rpc\Response\ReplaceFileSourceResponse;
+
+class OverloadMethodHandlerTest extends HandlerTestCase
+{
+    /**
+     * @var Reflector
+     */
+    private $reflector;
+
+    /**
+     * @var ObjectProphecy
+     */
+    private $overloadMethod;
+
+    public function setUp()
+    {
+        $this->reflector = Reflector::create(
+            new StringSourceLocator(SourceCode::fromString('<?php class ParentClass { public function foobar() {} }'))
```

Comment:
Because `reflectClassesIn` does not mean that classes can be located in the given source.